query/src/test/scala/filodb/query/exec/AggrOverRangeVectorsSpec.scala

```diff
@@ -353,7 +331,26 @@ class AggrOverRangeVectorsSpec extends FunSpec with Matchers with ScalaFutures {
     result6b.size shouldEqual 1
     result6b(0).key shouldEqual ignoreKey
     compareIter(result6b(0).rows.map(_.getDouble(1)), Seq(5.4d,5.6d).iterator)
+  }
+
+  it("should sum histogram RVs") {
```

Comment:
Can we have a few test cases where we add histograms with buckets that do not match?

Reply:
Sure.